FOCUS #242: Creating SKU ID based on work done by @arrarama (#255)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: supporting_content/dimensions/skuid.md

| Provider     | Data set                    | Column          |
| ------------ | --------------------------- | --------------- |
| AWS          | CUR                         | product/sku     |
| Azure        | Cost details export or API  | PartNumber      |
| GCP          | BigQuery Billing Export     | sku.id          |
| OCI          | Cost Reports                | cost/productSku |

## Example scenarios for current provider data

Current values observed in billing data for various scenarios:

| Provider     | Scenario                   | Pattern          |
| ------------ | -------------------------- | ---------------- |
| AWS          | CUR                        | FFNT87MQSCR328W6 |
| Azure        | Cost Details export or API | ABC-12345       |
| GCP          | BigQuery Billing Export    | F1ED-0732-5BDA   |
| OCI          | Cost Reports               | B91962           |

## Discussion / Scratch space

References

AWS - (<https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html#product-details-P>)

Azure - (<https://learn.microsoft.com/en-us/azure/cost-management-billing/automate/understand-usage-details-fields>)

> Big Query - (<https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage>)

Author: ```suggestion
GCP - (<https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage>)
(also, SKU information can be found via the BigQuery Pricing Export)
```

# Column: SKU ID

## Example provider mappings

Current column mappings found in available data sets:

| Provider     | Data set                    | Column          |
| ------------ | --------------------------- | --------------- |
| AWS          | CUR                         | product/sku     |
| Azure        | Cost details export or API  | PartNumber      |
| GCP          | BigQuery Billing Export     | sku.id          |
| OCI          | Cost Reports                | cost/productSku |

> |              |                             |                 |

Reviewer: Remove blank line

```suggestion

```

## Example scenarios for current provider data

Current values observed in billing data for various scenarios:

| Provider     | Scenario                   | Pattern          |
| ------------ | -------------------------- | ---------------- |
| AWS          | CUR                        | FFNT87MQSCR328W6 |
| Azure        | Cost Details export or API | ABC-12345       |
| GCP          | BigQuery Billing Export    | F1ED-0732-5BDA   |
| OCI          | Cost Reports               | B91962           |

> |              |                            |                  |

Reviewer: Remove blank line

```suggestion

```

# Column: SKU ID

## Example provider mappings 

Current column mappings found in available data sets:

| Provider     | Data set                    | Column          |
| ------------ | --------------------------- | --------------- |
| AWS          | CUR                         | product/sku     |
| Azure        | Cost details export or API  | PartNumber      |

> | GCP          | Big Query Export            | sku.id          |

Reviewer: ```suggestion
| GCP          | BigQuery Billing Export            | sku.id          |
```

| OCI          | Cost Reports                | cost/productSku |
|              |                             |                 |

## Example scenarios for current provider data

Current values observed in billing data for various scenarios:

| Provider     | Scenario                   | Pattern          |
| ------------ | -------------------------- | ---------------- |
| AWS          | CUR                        | FFNT87MQSCR328W6 |
| Azure        | Cost Details export or API |  ABC-12345       |

> | GCP          | Big Query                  | F1ED-0732-5BDA   |

Reviewer: ```suggestion
| GCP          | BigQuery Billing Export                  | F1ED-0732-5BDA   |
```
